data/json/items/book/maps.json

```diff
@@ -165,5 +165,19 @@
       ],
       "message": "You add roads and restaurants to your map."
     }
+  },
+  {
+    "id": "satellitemap",
+    "copy-from": "abstractmap",
+    "type": "GENERIC",
+    "name": { "str": "satellite map" },
+    "description": "This is a printed satellite map of the local area.  Due to it's low quality and lack of map legend, you are unable to recognise most buildings.  Using it will add terrain and roads to your map.",
```

Comment:
[basecamp_screenshots.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/6531286/basecamp_screenshots.zip)
Here are screenshoots of mansion and radio tower camps. Beds are built in separated steps, same goes for water well and root cellar. Fixing radio controls and fixing radio tower aren't shown, because those upgrades don't change map.